CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Protagonist Changelog
 
-## master
+## 2.0.0-pre.3
 
-### Bug fixes
+### Bug Fixes
 
-* Fixed segfault while parsing options 
+* Fixed a segfault while handling invalid options that contains unsupported
```

Comment:
I wanted to give some more emphasis that the segfault only happend with invalid option input as this affects less consumers of Protagonist.